pyeapi/api/ipinterfaces.py

```diff
@@ -222,7 +225,7 @@ def set_mtu(self, name, value=None, default=False, disable=False):
                 config to
 
             value (integer): The MTU value to set the interface to.  Accepted
-                values include 68 to 65535
+                values include IP_MTU_MIN to IP_MTU_MAX
```

Comment:
The doc string in pyeapi is used to build documentation, thus once rebuilt it's literally going to appear in the doc:  `values include IP_MTU_MIN to IP_MTU_MAX`, because the doc string does not interpolate values. So, here, I'm afraid it still must list the real numbers instead.

Reply:
As a compromise, I parenthesized the constants in the doc string.  I may not have time to make additional changes for a while.

Reply:
No worries, I'll push it myself then, as it seems a useful feature for the documentation.